Sming/Services/DateTime/DateTime.cpp

```diff
@@ -148,7 +149,14 @@ String DateTime::toFullDateTimeString()
 String DateTime::toISO8601()
 {
 	char buf[32];
-	m_snprintf(buf, sizeof(buf), _F("%02d-%02d-%02dT%02d:%02d:%02dZ"), Year, Month + 1, Day, Hour, Minute, Second);
+	m_snprintf(buf, sizeof(buf), _F("%04d-%02d-%02dT%02d:%02d:%02dZ"), Year, Month + 1, Day, Hour, Minute, Second);
+	return String(buf);
+}
+
+String DateTime::toHTTPDate()
+{
+    char buf[30];
```

Comment:
Fix the indentation please. Should be as the line below.

Reply:
Will do - IDE did its own thing. Will edit in text editor until I can whip IDE into line.